pkg/services/ssosettings/ssosettingsimpl/service_test.go

```diff
@@ -784,6 +795,50 @@ func TestService_ListWithRedactedSecrets(t *testing.T) {
 			want:    nil,
 			wantErr: true,
 		},
+		{
+			name: "should include saml provider if the provider is enabled and the licensing feature is enabled",
```

Comment:
I don't see how this test is doing what it is described in its title. It seems that below we are testing the `github` provider, not `SAML`.

Reply:
You're right. I probably added this test by mistake. I removed it.